uint32 GPADLEVEL_MOTORE=1
uint32 GPADLEVEL_MOTORD=2
uint32 GPADLEVEL_PUBFREQ=4
uint32 GPADLEVEL_BTN_TGLLIN=8
uint32 GPADLEVEL_BTN_TGLANG=16
uint32 GPADLEVEL_BTN_LOCK=32
uint32 GPADLEVEL_BTN_SFTBRK=64
uint32 GPADLEVEL_BTN_HRDBRK=128
uint32 GPADLEVEL_BTN_LINMODE=256
uint32 GPADLEVEL_BTN_ANGMODE=512
uint32 GPADLEVEL_AXIS_LIN=1024
uint32 GPADLEVEL_AXIS_LINSPEED=2048
uint32 GPADLEVEL_AXIS_ANG=4096
uint32 GPADLEVEL_AXIS_ANGSPEED=8192
uint32 GPADLEVEL_MAXLIN=16384
uint32 GPADLEVEL_MAXANG=32768
